src/data/pages/home/index.js

```diff
@@ -270,6 +270,140 @@ const data = {
       items: videoItems,
     },
   },
+  pt: {
+    hero: {
+      title:
+        'Programar dinamicamente o kernel para networking eficiente, observabilidade, rastreamento e segurança',
+      blackButtonTitle: 'Paisagem do Projeto',
```

Comment:
```suggestion
      blackButtonTitle: 'Cenário do Projeto',
```

Reply:
This case also needs a bit more complex and direct translation. I adopted the word "Scenario" because "Landscape" in Portuguese is commonly only used for natural images or architectural context.